packages/core/src/model.ts

```diff
@@ -56,7 +63,8 @@ type EventCreators<Self> = {
 };
 
 type ModelCreators<Self> = {
-  events: EventCreators<Prop<Self, 'events'>>;
+  events?: EventCreators<Prop<Self, 'events'>>;
+  actions?: EventCreators<Prop<Self, 'actions'>>;
```

Comment:
They are both using `EventCreators` - I guess their behavior is pretty much the same, right? We could rename the `EventCreators` to smth like `ModelCreators` though

Reply:
I added types for actions instead for now, even if they're the same. We can consolidate later, but I feel that they will be different, especially when we add guards, etc.